cypress/integration/google.spec.ts

```diff
@@ -0,0 +1,6 @@
+describe('This is my first cypress test', () => {
+  it('should have a title', () => {
+      cy.visit('https://www.google.com/');
+      cy.title().should('eq', 'Google');
+  });
+});
```

Comment:
When Github shows this icon.
![image](https://user-images.githubusercontent.com/10209180/112558850-b6221180-8d9d-11eb-816a-33516e7007f8.png)

It tells you that the file does not have the End Of File (EOF). This issue is solved by adding an empty line to the end of the file